### DISCUSSION TITLE
xch txid

### DISCUSSION THREAD

**Discussion by mahdibasir** [Chia-Network/chia-blockchain#16960] [Support]
my wallet number is 1143080087

I have transferred some xch to another wallet from Chia wallet and I need the TXID of this transaction to confirm the deposit.

thanks for this

**Comment by BrandtH22** [comment]
Hey @mahdibasir , the wallet fingerprint is tied directly to the keyset and is not something that will aid in identifying transactions onchain.

Generally the best way to get information regarding coin spends is to use a block explorer like [spacescan ](https://www.spacescan.io/) or [alltheblocks ](https://alltheblocks.net/). You will need to enter the xch address to which you sent the funds and should be able to find the coin spend that way :+1:
The transaction ID is generally going to be the coinID of the spent coin.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
